media/css/_faculty_structure.scss

```diff
@@ -0,0 +1,212 @@
+.bg-structure {
+  position: relative;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, height, margin, padding, position, width